packages/xo-web/src/common/tags.js

```diff
@@ -85,6 +108,26 @@ export default class Tags extends Component {
     event.preventDefault()
   }
 
+  _selectExistingTags = () => {
+    this._stopEdit()
```

Comment:
Maybe do it when the modal closes (resolves)?

Reply:
> So, If we keep it after closing modal, the user can't close the edition.

I don't understand this part.

Reply:
Sorry, I didn't understand for the first time and I confused it with the second comment.
So, I will stop the edition after closing modal.